### ISSUE TITLE
Issue with Example 1: 'repeat_report' not able to close/edit txt file

### ISSUE THREAD

**Issue by ifoxfoot** [quaquel/pyNetLogo#82]
Computer: 64-bit operating system, x64-based processor, Windows 10 Enterprise Version 21H2
netlogo: 6.3.0
pynetlogo: 0.5.1
python: 3.10

When I try to run `counts = netlogo.repeat_report(["count wolves", "count sheep"], 200, go="go")` from example 1 I get the following error:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\u4eewirf\Desktop\pynetlogo_practice\.conda\lib\site-packages\pynetlogo\core.py", line 557, in repeat_report
    os.remove(value)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\u4eewirf\\Desktop\\pynetlogo_practice\\wy_tx97t\\tmp0mnb41k5.txt'
```
`repeat_report` seems to correctly create a txt file, and to my knowledge it is not opened or being used by any process outside of pynetlogo. 

Thanks!

**Comment by quaquel** [comment]
Strange. My hunch is that it has to do with security settings of some sort. Are you running the code as administrator (or the browser in case of a notebook)?

**Comment by ifoxfoot** [comment]
Hi @quaquel thanks for the quick response. No, unfortunately I am not the administrator on this computer. Usually I do not have problems reading/writing files, but that could be the issue in this case!

**Comment by quaquel** [comment]
I tested my fix, and it seemed to work fine. Of course, a PR that fixes a new version of the same problem is always welcome.

**Comment by zatkins-dev** [comment]
I can confirm that @quaquel's fix doesn't solve #69 on Windows. I apologize for not coming back and checking that, I used this for a class project which ended prior to the update.

To clarify the issue: Windows requires that all handles to files be properly closed before the file can be deleted, unlike *nix systems which are more forgiving. Thus, since all that we want is to create a temp file and store the path to it, we should immediately close the file descriptor to make sure that it is free to be deleted later.

**Comment by zatkins-dev** [comment]
Specifically -- you call `open` on the filename again, so there's no reason to hold onto the file handle that `mkstemp` gives you.
